app/javascript/react/components/ProductShowContainer.js

```diff
@@ -1,42 +1,68 @@
-import React, { useState, useEffect } from 'react'
-import ProductShow from "./ProductShow"
+import React, { useState, useEffect } from "react";
+import ProductShow from "./ProductShow";
 
 const ProductShowContainer = (props) => {
   const [getProduct, setProduct] = useState({
     id: null,
     name: "",
     description: "",
     url: "",
-    image_url: ""
-  })
-  let productId = props.match.params.id
+    image_url: "",
+  });
+
+  const [reviewRecord, setReviewRecord] = useState([]);
```

Comment:
rather than reviewRecord, the getter/setter should be pluralized e.g. `[getReviews, setReviews]`